core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -998,6 +998,26 @@ package object config {
       .intConf
       .createWithDefault(Integer.MAX_VALUE)
 
+  private[spark] val SHUFFLE_SPILL_MAP_MAX_SIZE_FORCE_SPILL_THRESHOLD =
+    ConfigBuilder("spark.shuffle.spill.map.maxRecordsSizeForSpillThreshold")
```

Comment:
What does the "map" mean inside this config name?

Reply:
Why is it necessary to have different threshold between map task and reduce task?

Reply:
I have the same question. What's the use case to separate them differently?